ftprime/argrecorder.py

```diff
@@ -122,21 +125,25 @@ def tree_sequence(self, sites=None, mutations=None):
 
     def add_samples(self, samples, length, populations=None):
         '''
-        Add phony records that stand in for sampling the IDs in `samples`,
+        Set the `sample` flag on the samples,
+        and optionally set their populations.
+
+        Previously: Add phony records that stand in for sampling the IDs in `samples`,
         whose populations are given in `populations` (default: NULL), on a
         chromosome of total length `length`.
```

Comment:
Perhaps given https://github.com/jeromekelleher/msprime/pull/199 we _could_ just relabel existing nodes but the code is not currently doing that